src/widgets/main/ui/ExpoListContainer/index.tsx

```
@@ -1,33 +1,47 @@
-import { useMemo, useState } from 'react';
+import { useEffect, useState } from 'react';
 import { ExpoListItem } from '@/entities/main';
 import withLoading from '@/shared/hocs/withLoading';
-import { sortedData } from '@/shared/model/sortedData';
-import { ExpoItem } from '@/shared/types/main/type';
-import SortFilter from '@/shared/ui/SortFilter';
 import { filterOptions } from '../../constant/filterOptions';
 import { useExpoList } from '../../model/useExpoList';
 import EmptyExpoList from '../EmptyExpoList';
+import FormFilter from '../FormFilter';
+import { ExpoItem } from '@/shared/types/main/type';
```

ESLint

[error] `@/shared/types/main/type` import should occur before import of `../../constant/filterOptions`

(import/order)